service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -195,7 +196,12 @@ private ApiWorkspaceDescription buildWorkspaceDescription(
         .spendProfile(workspace.getSpendProfileId().map(SpendProfileId::getId).orElse(null))
         .stage(workspace.getWorkspaceStage().toApiModel())
         .gcpContext(gcpContext)
-        .azureContext(azureContext);
+        .azureContext(azureContext)
+        .lastUpdatedDate(
+            workspaceActivityLogDao
```

Comment:
Is there a reason to have the timezone mapping and null-ing here instead of inside the DAO?

Reply:
i think it's better to return optional than null in general but I can have it return offsetdate in the DAO instead of here.